docs/table/tips_and_tricks.rst

```diff
@@ -0,0 +1,37 @@
+Tips and Tricks
+***************
+
+Using QTable to store user-defined units in a FITS file
+=======================================================
+
+A `~astropy.table.QTable` can be used to store data as :ref:`user-defined
+units <astropy:defining_units>` in a FITS file.
+
+.. EXAMPLE START: Combining Units and Quantities
+
+To output data to the current directory::
+
+  >>> titter = u.def_unit('titter') 
```

Comment:
What is "titter"?

Reply:
I took one of the arbitrary defined units shown in a previous example and adapted it to this one.

Reply:
For official doc, would be less confusing if we stick to something closer to astronomy... 💭

Reply:
Yeah, makes sense. Gonna do those changes. Maybe I should just use something like "my_unit"?

And what about the CI exceptions? Should I ignore them?